mne_bids/mne_bids.py

```diff
@@ -485,30 +484,20 @@ def _sidecar_json(raw, task, manufacturer, fname, kind, eeg_ref=None,
     return fname
 
 
-def raw_to_bids(subject_id, task, raw_file, output_path, session_id=None,
-                acquisition=None, run=None, kind='meg', events_data=None,
-                event_id=None, hpi=None, electrode=None, hsp=None,
-                eeg_ref=None, eeg_gnd=None, config=None, overwrite=True,
-                verbose=True):
+def write_raw_bids(raw, bids_fname, output_path, kind='meg', events_data=None,
+                   event_id=None, eeg_ref=None, eeg_gnd=None, overwrite=True,
+                   verbose=True):
     """Walk over a folder of files and create BIDS compatible folder.
 
     Parameters
     ----------
-    subject_id : str
-        The subject name in BIDS compatible format ('01', '02', etc.)
-    task : str
-        Name of the task the data is based on.
-    raw_file : str | instance of mne.Raw
+    raw : instance of mne.Raw
         The raw data. If a string, it is assumed to be the path to the raw data
         file. Otherwise it must be an instance of mne.Raw
-    output_path : str
-        The path of the BIDS compatible folder
-    session_id : str | None
-        The session name in BIDS compatible format.
-    acquisition : str | None
-        Acquisition parameter for the dataset.
-    run : int | None
-        The run number for this dataset.
+    bids_fname : str
```

Comment:
I feel like this description doesn't really capture what you are expecting the user to enter

Reply:
+1 this is also problematic with regards to `copyfile_brainvision` and `copyfile_eeglab`, see below

Reply:
okay I changed it. Let me know if it looks better to you

Reply:
Looks better, but I would still throw in "expected" or "to be produced" or something to indicate that it is the file name of a file that doesn't exist before export.